synapse/handlers/room_member.py

```diff
@@ -25,8 +25,7 @@
 
 from twisted.internet import defer
 
-import synapse.server
-import synapse.types
+from synapse import types
```

Comment:
given that `types` is defined in the standard library, this doesn't seem like a great plan to me

Reply:
still not a great plan

Reply:
we're not importing stdlib `types` here, so we should be fine, we're not ghosting a module

although maybe we should rename the `synapse.types` to not potentially ghost the stdlib module